content/seismic/waves_at_interfaces.rst

```diff
@@ -28,11 +33,19 @@ Displacement of the earth from equilibrium position must be continuous across an
    = -1` means that the pulse will be reflected with a polarity change, for
    example at the rock-air interface, with an upward traveling wave.
 
-3. If   :math:`Z_2 >> Z_1`   :math:`R = 1`,  :math:`T = 0` (air earth
-   interface with downward traveling wave).
+3. If   :math:`Z_2 >> Z_1`   :math:`R = 1`,  :math:`T = 0` (wave travelling down through the air and hitting the air earth interface).
    
-We can illustrate these using the example of a wave propagating on a string. 
+We can illustrate scenarios two and three using the example of a wave propagating on a string. A string with a free end represents scenario two and a string with a fixed end represents scenario three.  These are illustrated in the following video
+
+.. raw:: html
+
+    <div style="margin-top:10px; margin-bottom:20px;" align="center">
+    <iframe width="560" height="315" src="https://www.youtube.com/embed/9OpL3OFuVXo" frameborder="0" allowfullscreen></iframe allowfullscreen>
+    </iframe>
+    </div>
 
+You can explore the concept further using an interactive web app `here <https://phet.colorado.edu/sims/html/wave-on-a-string/latest/wave-on-a-string_en.html>`__.
+    
 **Remark:**  Large amplitudes of the transmitted wave are sometimes counter-
```

Comment:
here, you can use a `.. note::` environment to have this highlighted eg. 

![image](https://cloud.githubusercontent.com/assets/6361812/19357320/00b2d492-9126-11e6-9cf4-1ea3d506be39.png)